### PR TITLE
CI: resolving Docs

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Debugging another force merge without waiting on CI
Resolves #15473

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7 @Felonious-Spellfire @tchaton @carmocca @akihironitta

### CODE REVIEW

**Comment by carmocca** [comment]
You won't see the docs fixes apply here until #15510 is merged

**Comment by yurijmikhalevich** [comment]
@carmocca, @Borda temporarily switch the trigger to `pull_request` which seem to enable us seeing the changes we may apply to CI running on this PR 🤔

**Comment by carmocca** [comment]
Ah didn't see that! Good. You'll just need to revert it before merging

File: docs/source-app/glossary.rst
Comment (Borda) [inline docs/source-app/glossary.rst:1]: @yurijmikhalevich is this removed on purpose?

Comment (yurijmikhalevich) [reply docs/source-app/glossary.rst:1]: @Borda, yes. This page is an unused duplicated of "basics.rst" in the same dir.

**Review by yurijmikhalevich — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Comment by williamFalcon** [comment]
Debugging another force merge without waiting on CI

🤣 
CI was green... GA actions were broken apparently

**Comment by Borda** [comment]
> > Debugging another force merge without waiting on CI
> 
> rofl CI was green... GA actions were broken apparently

Yes, I was wrong, built and tested #15486 and, as the last commit, added `pull_request_target`, which turned the check using master instead of the changes in the particular PR... so whatever we would do in any PR after #15486 merged would not be reflected in the PR's checks but also if it got broken on master it would not be visible as "fixed" in the PR...
see: https://github.com/Lightning-AI/lightning/pull/15508#issuecomment-1302418635

**Pull request merged**
